docs/UserGuide.adoc

```diff
@@ -67,11 +67,56 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
-=== Listing all persons : `list`
+=== Favoriting a person: `fav`
+
+Adds a person into a list of favorite contacts. +
+Format: `fav INDEX`
+
+****
+* Adds the person as a favorite contact at the specified `INDEX`.
+* The index refers to the index number shown in the most recent listing.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+* `list` +
+`fav 2` +
+Adds the 2nd person in the address book as a favorite contact.
+* `find Betsy` +
+`fav 1` +
+Adds the 1st person in the results of the `find` command as a favorite contact.
+
+=== Unfavoriting a person: `unfav`
+
+Removes a person from the list of favorite contacts. +
+Format: `unfav INDEX`
+
+****
+* Removes the person from the list of favorite contacts at the specified `INDEX`.
+* The index refers to the index number shown in the most recent listing.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+* `list` +
+`unfav 2` +
+Removes the 2nd person in the address book from the list of favorite contacts.
+* `find Betsy` +
+`unfav 1` +
+Removes the 1st person in the results of the `find` command from the list of favorite contacts.
+
+=== Listing all persons: `list`
 
 Shows a list of all persons in the address book. +
 Format: `list`
 
+=== Listing all favorite persons: `listfav`
+
+Shows a list of all favorite contacts in the address book. +
+Format: `listfav`
+
```

Comment:
can we extend the list command instead? for instance using `list fav` instead of adding a new command entirely

Reply:
Agreed! This is a better idea 👍 I'll update it ✅

Reply:
Has this been updated? @KeithSOC

Reply:
Updated ✔